### PR TITLE
Change the display random greeting feature to display random quote.

### PR DESCRIPTION
**Pull request by cl2112**

### CODE REVIEW

**Comment by cl2112** [comment]
PR 3 of 3.

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -7,3 +7,7 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.img {
+    width: 100%;
+}
```
Comment (ciciying) [inline portfolio/src/main/webapp/style.css]: Please add a new line at the end of this file.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,7 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.img {
+    width: 100%;
```
Comment (ciciying) [inline portfolio/src/main/webapp/style.css]: Please use consistent indentation, (either 2-space or 4-space indentation but make sure it's uniform across this file)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,24 @@
...
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/** 
+ * Adds a random quote from the TV show Parks and Rec to the page.
+*/
```
Comment (ciciying) [inline portfolio/src/main/webapp/script.js]: nit: make sure your block comment is aligned correctly

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,17 +12,24 @@
...
+      "I typed your symptoms into the thing up here and it says you could " + 
+        "have network connectivity problems.",
+      "Ron: I'll have the number 8. Waiter: That's a party platter. It " + 
+        "serves 12 people. Ron: I know what I'm about, son.",
+      "I'm allergic to sushi. Everytime I eat more than 80 pieces I throw up.",
+  ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random quote.
```
Comment (ciciying) [inline portfolio/src/main/webapp/script.js]: Also make sure you're using the same indentation across this file. (and below)

**Review by ciciying — APPROVED** [review]
LGTM, just some indentation fixes and small nits from me.

**Comment by cl2112** [comment]
Thank you for the comments. I have made the changes and I am going to merge and then close this branch.

**Pull request merged**